cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/LoadCSVTest.scala

```diff
@@ -151,6 +152,7 @@ For more information, see <<query-using-periodic-commit-hint>>.
       queryText = s"USING PERIODIC COMMIT LOAD CSV FROM '%ARTIST%' AS line CREATE (:Artist {name: line[1], year: toInteger(line[2])})",
       optionalResultExplanation = "",
       assertions = (p) => assertStats(p, nodesCreated = 4, propertiesWritten = 8, labelsAdded = 4))
+    Note.apply(Paragraph("The <<query-use, `USE` clause>> cannot be used together with the `PERIODIC COMMIT` clause."))
```

Comment:
```
<<query-use, `USE` clause>>
```
Having the backticks on `USE` makes it look like the link is just the word `clause`. Would it be better to not have backticks on `USE` so the whole thing looks like a link or keep it as it is?